library/evaluation/classification.py

```diff
@@ -12,15 +12,15 @@
 from type import Evaluators, PredsWithProbs
 
 
-def __wrap_sklearn_scorer(scorer: Callable) -> Callable:
+def __wrap_sklearn_scorer(scorer: Callable, *args, **kwargs) -> Callable:
     def wrapper(y_true, predicted_probs: List[PredsWithProbs]) -> float:
-        return scorer(y_true, [item[0] for item in predicted_probs])
+        return scorer(y_true, [item[0] for item in predicted_probs], *args, **kwargs)
 
     return wrapper
 
 
 classification_metrics: Evaluators = [
-    ("f1", __wrap_sklearn_scorer(f1_score)),
+    ("f1", __wrap_sklearn_scorer(f1_score, average="micro")),
```

Comment:
average : {'micro', 'macro', 'samples', 'weighted', 'binary'} or None, \
            default='binary'
        This parameter is required for multiclass/multilabel targets.
        If ``None``, the scores for each class are returned. Otherwise, this
        determines the type of averaging performed on the data:

        ``'binary'``:
            Only report results for the class specified by ``pos_label``.
            This is applicable only if targets (``y_{true,pred}``) are binary.
        ``'micro'``:
            Calculate metrics globally by counting the total true positives,
            false negatives and false positives.
        ``'macro'``:
            Calculate metrics for each label, and find their unweighted
            mean.  This does not take label imbalance into account.
        ``'weighted'``:
            Calculate metrics for each label, and find their average weighted
            by support (the number of true instances for each label). This
            alters 'macro' to account for label imbalance; it can result in an
            F-score that is not between precision and recall.
        ``'samples'``:
            Calculate metrics for each instance, and find their average (only
            meaningful for multilabel classification where this differs from
            :func:`accuracy_score`).